### DISCUSSION TITLE
Severity Score - incorrect

### DISCUSSION THREAD

**Discussion by ragufronz** [aquasecurity/trivy#4600] [Q&A] [answered]
### Description

JSON Reports generated with a scan often contain severity scores that are lower than the official scores reported on the NVD Website (https://nvd.nist.gov). This seems to happen if there are multiple severity scores in the attribute CVSS. It seems that the last reported score in the list is used for calculating the severity level.

### Desired Behavior

If there are multiple scores being reported for a CVE, the highest of these scores (and not the last one in the list) should be used for calculating the "Severity" Value being sent back in the JSON report.

### Actual Behavior

Severity Scores (reported under Vulnerabilities -> Severity) are often not the same as those reported on the NVD Website. This seems to happen when the "CVSS" Attribute contains more than one severity score.

Ex: if the attribute has the score 7.5 and 5.9, and 5.9 appears as the last reported score in the list, the severity level is shown as MEDIUM instead of HIGH.

### Reproduction Steps

```bash
1. Run a report that contains the CVE CVE-2023-2650 and has more than one score in CVSS. Ex: for the official Docker Hub Container (tomcat:9.0.75-jre17-temurin-jammy)
2. CVSS contains the scors 7.5 and 5.9
3. NVD Database reports severity score of HIGH
4. Trivy reports severity score of MEDIUM
```

### Target

Container Image

### Scanner

Vulnerability

### Output Format

JSON

### Mode

Standalone

### Debug Output

```bash
Not requred
```

### Operating System

Ubuntu

### Version

```bash
aquasec/trivy:latest
```

### Checklist

- [ ] Run `trivy --reset`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by knqyf263** [comment, marked as answer]
The severity from vendors takes precedence over NVD.
https://aquasecurity.github.io/trivy/v0.42/docs/scanner/vulnerability/os/#data-source-selection
